dali/python/nvidia/dali/plugin/pytorch/__init__.py

```diff
@@ -795,3 +853,266 @@ def __next__(self) -> List[Dict[str, torch.Tensor]]:
     DENSE_TAG: str = "dense"
     SPARSE_LIST_TAG: str = "sparse_list"
     SPARSE_COO_TAG: str = "sparse_coo"
+
+
+class DALIProxy:
```

Comment:
Suggestion: maybe we should extract DALI Proxy implementation to a separate file? We can still make it importable from `dali.plugin.pytorch` by importing it here in `__init__.py`. However, extracting all those things to dedicated submodule `dali.plugin.pytorch.proxy` seems even cleaner